### PR TITLE
Update codelab 'Pub/Sub to CF'

### PR DESCRIPTION
**Pull request by fifthposition**
Use new artifacts UI. Also add missing attribute constraint required for the Google Cloud Storage trigger to work.

### CODE REVIEW

File: guides/tutorials/codelabs/pubsub-to-cf/index.md [hunk 1]
```diff
@@ -4,10 +4,12 @@
  sidebar:
    nav: guides
 ---
-
- {% include alpha version="1.10 and later" %}
  
  {% include toc %}
+
+> This codelab assumes that you have enabled the `artifactsRewrite` feature flag. In `~/.hal/$DEPLOYMENT/profiles/settings-local.js` (where `$DEPLOYMENT` is typically `default`), add:
```
Comment (dorbin) [inline guides/tutorials/codelabs/pubsub-to-cf/index.md:10]: As before, we'll need a way to make sure to remove this if the feature becomes the default and this flag becomes moot.

File: guides/tutorials/codelabs/pubsub-to-cf/index.md [hunk 2]
```diff
@@ -102,31 +104,29 @@ e. Apply your changes:
 
 ## 5. Configure the Application and Pipeline
 
-a. Create a new pipeline for your application. In the pipeline configuration, configure the Expected Artifacts as shown below, a GCS object with the path to the application archive in your GCS bucket:
+a. Create a new pipeline for your application. In the pipeline configuration, under Automated Triggers, add a new trigger of **Type** "Pub/Sub". Select GCP Cloud Pub/Sub as the **Pub/Sub System Type**, select your GCP Cloud Pub/Sub Spinnaker subscription in **Subscription Name**, and under **Attribute Constraints**, add an entry with the key `eventType` and value `OBJECT_FINALIZE ` (see the [Google Cloud Storage documentation](https://cloud.google.com/storage/docs/pubsub-notifications)).
```
Comment (dorbin) [inline guides/tutorials/codelabs/pubsub-to-cf/index.md]: Can you break this out into a list, or even a table?

File: guides/tutorials/codelabs/pubsub-to-cf/index.md [hunk 3]
```diff
@@ -102,31 +104,29 @@ e. Apply your changes:
...
-b. Configure Automated Triggers, adding the GCP Cloud Pub/Sub Spinnaker subscription as a trigger and selecting the Artifact Constraints:
+b. In the **Artifact Constraints** dropdown, select "Define a new artifact..." to bring up the **Expected Artifact** form. Enter a **Display Name** or keep the auto-generated default. Select your GCS account in the **Account** dropdown and enter the path to the artifact in the **Object path** field.
 
   {% include figure
-     image_path="./triggers.png"
+     image_path="./expected-artifact.png"
   %}
 
-c. Add a Deploy stage to the pipeline. Create a new server group and provide details on deployment settings, the application artifact, and the manifest artifact:
+Click **Save Artifact**.
```
Comment (dorbin) [inline guides/tutorials/codelabs/pubsub-to-cf/index.md]: This should get a number (letter). In fact, I think you can replace all the letters with with `a` in each case, and it should order them as it does with numbers.

**Review by dorbin — COMMENTED** [review]
Couple of small comments. LGTM, but I'll wait to approve so nobody merges it yet.

File: guides/tutorials/codelabs/pubsub-to-cf/index.md (hunk 1 above)
Comment (fifthposition) [reply guides/tutorials/codelabs/pubsub-to-cf/index.md:10]: I'm happy to go through in that case and at that time, search for the flag, and remove mentions like this. I only want it for now, while it's necessary in order to see the new UI.

File: guides/tutorials/codelabs/pubsub-to-cf/index.md (hunk 2 above)
Comment (fifthposition) [reply guides/tutorials/codelabs/pubsub-to-cf/index.md]: Good suggestion--thanks! Made a list here (and also for the artifact configuration).

File: guides/tutorials/codelabs/pubsub-to-cf/index.md (hunk 3 above)
Comment (fifthposition) [reply guides/tutorials/codelabs/pubsub-to-cf/index.md]: I seemed to recall having tried that and not being able to get it to work, but I did add a letter. Thank you!

**Comment by fifthposition** [comment]
Addressed feedback (thanks!) @dorbin.

**Review by dorbin — APPROVED** [review]
LGTM
Added @jkschneider as tech reviewer

**Review by Jammy-Louie — APPROVED** [review]
LGTM

**Pull request merged**